nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTests.Sdk.cs

```diff
@@ -186,6 +187,7 @@ await TestUpdateForProject("Some.Package", "12.0.1", "13.0.1",
                 projectContents: $"""
                     <Project Sdk="Microsoft.NET.Sdk">
                       <PropertyGroup>
+                        <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
```

Comment:
The change to force the ruby tests for C# to use the version of the .NET SDK in `global.json` for NuGetUpdater cause an error that package versions shouldn't be used in projects using central package management. I think this is because the build then picks up the `Directory.Packages.props` file in that directory despite the test project not being in the directory tree of that file.